python-avocado.spec

```diff
@@ -185,7 +185,6 @@ PATH=%{buildroot}%{_bindir}:%{buildroot}%{_libexecdir}/avocado:$PATH \
 %{_bindir}/avocado-runner
 %{_bindir}/avocado-runner-noop
 %{_bindir}/avocado-runner-dry-run
-%{_bindir}/avocado-runner-exec
```

Comment:
it's a good idea  to add a `%changelog` entry for this.

Reply:
Sure, I forgot those.